languages/fr-fr.js

```diff
@@ -0,0 +1,181 @@
+  CONFIRM_ACCOUNT_HEADER_INVALID: `${_CC.config.siteTitle} | Lien de confirmation invalide`,
+  CONFIRM_ACCOUNT_HEADER_VALID: `${_CC.config.siteTitle} | Confirmer le compte`,
+  CONFIRM_ACCOUNT_INVALID: "Ce lien de confirmation n'est pas valide, peut-être que le compte a été supprimé ou que certains caractères à la fin ont été enlevés?",
+  CONFIRM_ACCOUNT_SET_PW_BUTTON: `Rejoindre ${_CC.config.siteTitle}`,
+  CONFIRM_ACCOUNT_SET_PW_PLACEHOLDER: 'pa$$word!',
+  CONFIRM_ACCOUNT_SET_PW_TEXT: name => `Bonjour ${name}! Veuillez taper votre mot de passe ici.`,
+  CONFIRM_ACCOUNT_SUCCESS: `Bienvenue à ${_CC.config.siteTitle}!`,
+  LOGIN_BUTTON: 'Connexion',
+  LOGIN_PASSWORD_PLACEHOLDER: 'pa$$word!',
+  LOGIN_PASSWORD: 'Password',
+  LOGIN_USERNAME_PLACEHOLDER: 'jean',
+  LOGIN_USERNAME: `Nom d'utilisateur`,
+  LOGOUT_BUTTON: 'Deconnexion',
+  NAVBAR_ADMIN: 'Paramètres Administrateur',
+  NAVBAR_LOGIN: 'Connexion',
+  NAVBAR_LOGOUT: 'Deconnexion',
+  NAVBAR_PROFILE: 'Profile',
+  NAVBAR_WISHLIST: `Ma liste d'envies`,
+  NOTE_BACK: name => `Retour à la liste d'envie de ${name}`,
+  NOTE_GET_PRODUCT_DATA: 'Obtenir des données sur le produit',
+  NOTE_GUARD: 'Utilisateur invalide',
+  NOTE_IMAGE_URL: `URL de l'Image`,
+  NOTE_MISSING_PROP: prop => `Propriété manquante ${prop}`, // not really possible to localize this unfortunately
+  NOTE_NAME: 'Nom',
+  NOTE_NOTE: 'Note',
+  NOTE_PRICE: 'Prix',
+  NOTE_REFRESH_DATA: 'Actualiser les données',
+  NOTE_REMOVE_GUARD: 'Utilisateur invalide',
+  NOTE_REMOVE_MISSING: `N'a pas de note`,
+  NOTE_REMOVE_SUCCESS: 'Note supprimée avec succès',
+  NOTE_SAVE_BUTTON: `Enregistrer l'article`,
+  NOTE_SUCCESS: 'Enregistré avec succès!',
+  NOTE_URL: 'URL',
+  PROFILE_HEADER: 'Profile',
+  PROFILE_PASSWORD_BUTTON: 'Sauvegarder',
+  PROFILE_PASSWORD_NEW: 'Nouveau Mot de Passe',
+  PROFILE_PASSWORD_OLD_MISMATCH: 'Ancien mot de passe incorrect',
+  PROFILE_PASSWORD_OLD: 'Ancien mot de passe',
+  PROFILE_PASSWORD_PLACEHOLDER: 'pa$$word!',
+  PROFILE_PASSWORD_REQUIRED_NEW: 'Un nouveau mot de passe est requis',
+  PROFILE_PASSWORD_REQUIRED_OLD: `L'ancien mot de passe est requis`,
+  PROFILE_PASSWORD_SUCCESS: 'Les modifications on été enregistré avec succès!',
+  PROFILE_PASSWORD_TITLE: name => `Paramètres de profil - Mot de passe - ${name}`,
+  PROFILE_PFP_IMAGE_URL: `URL de l'Image`,
+  PROFILE_SAVE_PFP_DISABLED: 'Les photos de profil sont désactivées.',
+  PROFILE_SAVE_PFP_SUCCESS: 'Photo de profil enregistrée !',
+  PROFILE_SECURITY_CHANGE_PASSWORD: 'Changer le mot de passe',
+  PROFILE_SECURITY: 'Sécurité',
+  PROFILE_TITLE: name => `Paramètres de profil - ${name}`,
+  RESET_PASSWORD_BUTTON: 'Réinitialiser le mot de passe',
+  RESET_PASSWORD_GREETING_EXPIRED: 'Votre lien de réinitialisation a expiré. Veuillez en demander un nouveau.',
+  RESET_PASSWORD_GREETING_INVALID: "Ce lien de réinitialisation n'est pas valide, peut-être le lien a-t-il expriré ou certains caractères à la fin ont-ils été enlevés?",
```

Comment:
"Ce lien de réinitialisation n'est pas valide, peut-être le lien a-t-il expiré ou a été tronqué ?"

Reply:
'expiré' instead of 'expriré'